lib/auth/auth.go

```diff
@@ -2004,11 +2004,15 @@ func (a *Server) GetClusterID(ctx context.Context, opts ...services.MarshalOptio
 // GetAnonymizationKey returns the anonymization key that identifies this client.
 // It falls back to the cluster ID if the anonymization key is not set in license file.
```

Comment:
Should we update this comment? Maybe:
```suggestion
// The anonymization key may be any of the following, in order of precedence:
// - (Teleport Cloud) a key provided by the Teleport Cloud API
// - a key embedded in the license file
// - the cluster's UUID
```